src/helpers/validateTxXdr.ts

```diff
@@ -2,7 +2,7 @@ import trim from "lodash/trim";
 import { xdr } from "stellar-sdk";
 import validateBase64 from "./validateBase64";
 
-export default function validateTxXdr(input, networkPassphrase) {
+export const validateTxXdr = (input: string) => {
```

Comment:
Went ahead and did the ts conversion because it was low lift and it's easier to stub a named method rather than a default method.